automation/submission-perf-inc.sh

```diff
@@ -244,16 +255,33 @@ monitorSubmission() {
 
     getAccessToken "$user"
 
-    # curl -s -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/submissions/queueStatus" | jq -r '"\(now),\(.workflowCountsByStatus.Queued),\(.workflowCountsByStatus.Running),\(.workflowCountsByStatus.Submitted)"' | tee -a workflow-progress-$BUILD_NUMBER.csv
-    submissionDetails=$(curl \
-        -X GET \
-        --header 'Accept: application/json' \
-        --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-        "https://api.firecloud.org/api/workspaces/$namespace/$name/submissions/$submissionId")
+    printf "\nFetching status for submission ID '%s':" "${submissionId}"
 
-    submissionStatus=$(jq -r '.status' <<< "${submissionDetails}" )
-    workflowsStatus=$(jq -r '.workflows[] | .status' <<< "${submissionDetails}")
-    workflowFailures=$(jq -r '[.workflows[] | select(.status == "Failed")] | length' <<< "${submissionDetails}")
+    # curl -s -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/submissions/queueStatus" | jq -r '"\(now),\(.workflowCountsByStatus.Queued),\(.workflowCountsByStatus.Running),\(.workflowCountsByStatus.Submitted)"' | tee -a workflow-progress-$BUILD_NUMBER.csv
```

Comment:
Since it's unused (and has no context), now might be a good time to just remove this comment?